internal/services/kusto/kusto_cluster_resource_test.go

```diff
@@ -856,7 +856,7 @@ resource "azurerm_network_security_group" "test" {
 
 resource "azurerm_network_security_rule" "test_allow_management_inbound" {
   name                        = "AllowAzureDataExplorerManagement"
-  priority                    = 106
+  priority                    = 1000
```

Comment:
After consulting with kusto team, we change this priority because the original 106 priority will conflict with Microsoft NRMS rules when testing locally inside Microsoft network and cause diff after perform terraform refresh. 
BTW, any number larger than 200 is ok here to avoid conflict with internal rules.